kernel/arch/dreamcast/include/dc/maple.h

```diff
@@ -192,6 +192,8 @@ typedef struct maple_frame {
 #endif
 } maple_frame_t;
 
+#define MAPLE_FRAME_WORD_BYTES  4  /**< \brief Bytes within a single word of a frame */
```

Comment:
Is this something that should be its own define? It seems that a 'sizeof(uint32_t)' might be sufficient to be descriptive? The word size is also described in maple_frame_t->length. That's really minor tho. Seems we don't do that anywhere in /arch/dreamcast/. Just something to ponder.